components/postmark/actions/send-single-email/send-single-email.mjs

```diff
@@ -1,92 +1,50 @@
 // legacy_hash_id: a_8KiGrJ
-import { axios } from "@pipedream/platform";
+import postmark from "../../postmark.app.mjs";
+
+let objSharedProps = {};
+postmark.methods.listSharedProps().forEach((propName) => {
+  objSharedProps[propName] = {
+    propDefinition: [
+      postmark,
+      propName,
+    ],
+  };
+});
 
 export default {
   key: "postmark-send-single-email",
-  name: "Send an Email with Postmark to a Single Recipient",
-  description: "Send an Email with Postmark to a Single Recipient",
-  version: "0.1.1",
+  name: "Send an email",
+  description: "Send an email with Postmark",
+  version: "0.1.4",
```

Comment:
Typically the version would be bumped by one `patch` version, to `0.1.2`

But in this case since it has some kind of structural changes, I think we could bump the `minor` version to `0.2.0`. See [docs](https://pipedream.com/docs/components/guidelines/#versioning)

```suggestion
  version: "0.2.0",
```

Reply:
Good reminder. I increased the version numbers as needed to update the component in early development, but will avoid committing such changes and make sure the version is updated/created to/at the correct number on a pull request.